types.nix

```diff
@@ -164,6 +164,10 @@ rec {
         umount -Rlv /mnt
       fi
 
+      for ds in $(zpool list -H -o name); do
+        zpool destroy "$ds"
+      done
```

Comment:
```suggestion
      if command -v zpool; then
        for ds in $(zpool list -H -o name); do
          zpool destroy "$ds"
        done
      fi
```

Reply:
`s/command -v/type -p/g` because it's shorter :-p